projectroles/templatetags/projectroles_tags.py

```diff
@@ -68,7 +68,7 @@ def has_star(project, user):
     """Return True/False for project star tag state"""
     return user.has_perm(
         'projectroles.view_project', project
-    ) and get_tag_state(project, user, PROJECT_TAG_STARRED)
```

Comment:
The original implementation of this looked a bit weird to me, so I had a closer look. Turns out the `has_star()` template tag is no longer used for anything nowadays. I think this is residue from a time when the site had a server side project list.

This template tag and tests related to it can be removed.